appserver/common/stats77/pom.xml

```diff
@@ -72,5 +73,11 @@
             <artifactId>internal-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.management.j2ee</groupId>
+            <artifactId>javax.management.j2ee-api</artifactId>
+            <version>${javax.management.j2ee-api.version}</version>
```

Comment:
Did you consider
```
    <groupId>jakarta.management.j2ee</groupId>
    <artifactId>jakarta.management.j2ee-api</artifactId>
```

Reply:
javax.management is already in Payara, switching to the Jakarta version is a separate issue.

Reply:
As it is managed in https://github.com/payara/Payara/blob/13bd97b3cd8a0c9d3c51fe24eef40f50f93d03b4/appserver/pom.xml#L1006-L1010 could the version element be skipped here then?